### PR TITLE
Show permissions for groups and users

### PR DESCRIPTION
**Pull request by odinuge**
Screenshot from 2020-02-29 15-59-03
Screenshot from 2020-02-29 15-58-47

### CODE REVIEW

**Comment by odinuge** [comment]
Depends on https://github.com/webkom/lego/pull/1798

**Comment by odinuge** [comment]
Updated with group names:

Screenshot from 2020-02-29 16-23-50

File: app/routes/users/components/UserProfile.js [hunk 1]
```diff
@@ -312,6 +312,66 @@ export default class UserProfile extends Component<Props, EventsProps> {
...
+                                abakusGroup.id
+                              }/permissions/`}
+                            >
+                              {' '}
+                              {abakusGroup.name}{' '}
+                            </Link>
+                          </h4>
+                          <ul>
+                            {permissions.map(permission => (
+                              <li key={permission + abakusGroup}>
```
Comment (SmithPeder) [inline app/routes/users/components/UserProfile.js]: string + object 🙃 
```
const stuff = "perm" + {"id": 1, "name": "name"};
undefined
suff
"[object Object]perm"
```

Guess I'll JS then 👍

File: app/routes/users/components/UserProfile.js [hunk 2]
```diff
@@ -312,6 +312,66 @@ export default class UserProfile extends Component<Props, EventsProps> {
...
+                        <>
+                          <h4>
+                            Rettigheter fra
+                            <Link
+                              to={`/admin/groups/${
+                                abakusGroup.id
+                              }/permissions/`}
+                            >
+                              {' '}
+                              {abakusGroup.name}{' '}
```
Comment (SmithPeder) [inline app/routes/users/components/UserProfile.js]: need the space this {' '} creates?

File: app/routes/users/components/UserProfile.js [hunk 3]
```diff
@@ -312,6 +312,66 @@ export default class UserProfile extends Component<Props, EventsProps> {
                 </Card>
               </div>
             )}
+            {/* canChangeGrade is a good heuristic if we should show permissions. All users can see their own permission via the API, but only admins can show permissions for other users.*/}
```
Comment (SmithPeder) [inline app/routes/users/components/UserProfile.js]: newline after period ⭕️  this won't fit in my terminal 💯

File: app/routes/admin/groups/components/GroupPermissions.js [hunk 4]
```diff
@@ -24,24 +31,75 @@ const removePermission = (permission, group, editGroup) =>
...
+  const nestedPermissionsList = nestedPermissions
+    .map(
+      ({ abakusGroup, permissions }) =>
+        !!permissions.length && (
+          <>
+            <h4>
+              Rettigheter fra
+              <Link to={`/admin/groups/${abakusGroup.id}/permissions/`}>
+                {' '}
+                {abakusGroup.name}{' '}
```
Comment (SmithPeder) [inline app/routes/admin/groups/components/GroupPermissions.js]: need the space this {' '} creates?

File: app/routes/users/components/UserProfile.js (hunk 1 above)
Comment (SmithPeder) [reply app/routes/users/components/UserProfile.js]: If the object always gets parsed to _"[object Object]"_  will just be the same as setting key `key={permission}`?

Comment (SmithPeder) [reply app/routes/users/components/UserProfile.js]: or set it as _abakusGroup.id_ or _abakusGroup.name_ ?

File: app/routes/admin/groups/components/GroupPermissions.js (hunk 4 above)
Comment (odinuge) [reply app/routes/admin/groups/components/GroupPermissions.js]: Nop, thx

File: app/routes/users/components/UserProfile.js (hunk 3 above)
Comment (odinuge) [reply app/routes/users/components/UserProfile.js]: :100:

File: app/routes/users/components/UserProfile.js (hunk 1 above)
Comment (odinuge) [reply app/routes/users/components/UserProfile.js]: Good catch! Used `abakusGroup = pk` at first, but forgot to change this. Thx

File: app/routes/users/components/UserProfile.js (hunk 2 above)
Comment (odinuge) [reply app/routes/users/components/UserProfile.js]: nop

**Pull request merged**